test/e2e/installplan_e2e_test.go

```diff
@@ -2998,6 +3000,171 @@ var _ = Describe("Install Plan", func() {
+				},
+			}
+			Expect(ctx.Ctx().Client().Create(context.Background(), catsrc)).To(Succeed())
+
+			// Wait for the CatalogSource to be ready
+			catsrc, err := fetchCatalogSourceOnStatus(crc, catsrc.GetName(), catsrc.GetNamespace(), catalogSourceRegistryPodSynced)
+			Expect(err).ToNot(HaveOccurred())
+
+			catsrcName = catsrc.GetName()
+
+			// Create the OperatorGroup
+			og := &operatorsv1.OperatorGroup{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "og",
+					Namespace: ns.GetName(),
+				},
+				Spec: operatorsv1.OperatorGroupSpec{
+					TargetNamespaces: []string{ns.GetName()},
+				},
+			}
+			Expect(ctx.Ctx().Client().Create(context.Background(), og)).To(Succeed())
+
+			now := metav1.Now()
+			ip = &operatorsv1alpha1.InstallPlan{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "ip",
+					Namespace: ns.GetName(),
+				},
+				Spec: operatorsv1alpha1.InstallPlanSpec{
+					ClusterServiceVersionNames: []string{"foobar"},
+					Approval:                   v1alpha1.ApprovalAutomatic,
+					Approved:                   true,
+				},
+				Status: operatorsv1alpha1.InstallPlanStatus{
+					Phase:          operatorsv1alpha1.InstallPlanPhaseInstalling,
+					CatalogSources: []string{},
+					BundleLookups: []operatorsv1alpha1.BundleLookup{
+						{
+							Identifier: "foobar.v0.0.1",
+							CatalogSourceRef: &corev1.ObjectReference{
+								Namespace: ns.GetName(),
+								Name:      catsrcName,
+							},
+							Conditions: []operatorsv1alpha1.BundleLookupCondition{
+								{
+									Type:               operatorsv1alpha1.BundleLookupPending,
+									Status:             corev1.ConditionTrue,
+									Reason:             "JobIncomplete",
+									Message:            "unpack job not completed",
+									LastTransitionTime: &now,
+								},
+							},
+						},
+					},
+				},
+			}
+		})
+
+		AfterEach(func() {
+			Expect(ctx.Ctx().Client().Delete(context.Background(), ns)).To(Succeed())
+		})
+
+		It("should show an error on the bundlelookup condition for a non-existent bundle image", func() {
+			// Create an InstallPlan status.bundleLookups.Path specified for a non-existent bundle image
+			ip.Status.BundleLookups[0].Path = "quay.io/foo/bar:v0.0.1"
+
+			// We wait for some time over the bundle unpack timeout (i.e ActiveDeadlineSeconds) so that the Job can eventually fail
+			// Since the default --bundle-unpack-timeout=10m, we override with a shorter timeout via the
+			// unpack timeout annotation on the InstallPlan
+			annotations := make(map[string]string)
+			annotations[bundle.BundleUnpackTimeoutAnnotationKey] = "1m"
+			ip.SetAnnotations(annotations)
+			waitFor := 1*time.Minute + 30*time.Second
+
+			outIP := ip.DeepCopy()
+
+			Expect(ctx.Ctx().Client().Create(context.Background(), outIP)).To(Succeed())
+
+			// The status gets ignored on create so we need to update it else the InstallPlan sync ignores
+			// InstallPlans without any steps or bundle lookups
+			outIP.Status = ip.Status
```

Comment:
Is this plus the above clone necessary? There are other InstallPlan tests that call update and status update consecutively that seem to work, which makes me think that Create() doesn't overwrite the status of the provided object.

Reply:
Hmm I could have sworn that create [updates the object](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/vendor/sigs.k8s.io/controller-runtime/pkg/client/typed_client.go#L51) but from a quick test the status block does not get updated.

So yeah no need for a clone in that case.